src/plugins/notebook/components/Notebook.vue

```diff
@@ -129,6 +133,13 @@
                     To start a new entry, click here or drag and drop any object
                 </span>
             </div>
+            <div
+                v-if="isRestricted && (selectedPage && selectedPage.isLocked)"
```

Comment:
Is it implicit that if a page is locked, it is also restricted? Do we need the isRestricted check here?

Reply:
Yeah I think we could probably do away with that. Good catch!